website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,187 @@
+- **OPTIONALLY** setting the resources limit for Memory and CPU for the Operator Pod and any `Pod/Deployment` managed by it.
+**NOTE** If your project is integrated with OLM then, admins would be able to customize it for the `Operator/manager` 
+via the [subscription][olm-subscriptions]). Also, be aware that if the project was generated by Kubebuilder or SDK scaffold 
+then some values for the `Operator/manager`(See `config/manager/manager.yaml`) 
+are populated by default to get you started, however, you ought to optimize them based on your own tests and the specific needs of your operator.
+- **Should** provide the mechanisms for [Monitoring compute & memory resource usage][k8s-metrics]. Kubebuilder and SDK provide some [metrics][kb-metrics] 
+which can help you with. Note that cluster admins can use it to monitor and resize the Operator and its Operands.
+**NOTE**  If the Operator is integrated with OLM and the bundle has a `PodMonitor` or a `ServiceMonitor` the 
+complete `InstallPlan` will fail on a cluster, which does not have these CRD/the Prometheus operator installed. 
+In this case, you might want to ensure the dependency requirement with [OLM dependency][olm-dep] or make clear 
+its requirement for the Operator consumers. 
+- **Should** allow admins to customize the `requests/limits` amounts defined for the `Pod/Deployment`
+created by the Operator and not keep them fixed on the code.
+- **Should** document how your Operator consumer can customize/rightsize the resources requests and limits for the 
+Operator and Operands `Pod/Deployments` or let its consumers know that the solution could adopt a better approach where 
+the Operator automatically adjust the values to the environment rather than asking its consumers to amend them
+
+You might also consider leveraging the Vertical Pod Autoscaler to have the resources requested by the Operator 
+automatically adjusted to the cluster where it is deployed.[More info][k8s-vertical-pod-scaling]. You might also 
+look at allowing [horizontal pod autoscaling][k8s-autoscaling] for the other `Pod/Deployments` created by your project. 
+In this case, be aware that resource requests are also required for horizontal pod autoscaling to work.
+
+**NOTE:** If you are using OpenShift as Kubernetes distribution you might want to check the 
+doc [Automatically adjust pod resource levels with the vertical pod autoscaler
+][ocp-pod-scaling]. Caveat: If the VPA CRD is not available in the cluster where the operator 
+gets deployed the InstallPlan will fail. PLease, see how to work with [OLM dependency][olm-dep] 
+if your project integrating with.
+
+### But, what happens
+
+#### when the resource requests are not set?
+
+**If not set:** 
+- configurations made by the cluster administrators such as [ResourceQuota][k8s-resources-quotas] might not work with the usage of the LimitRanger admission controller to force defaults might not work.
```

Comment:
This sentence is confusing. So what exactly "might not work"? The ResourceQuota? or the LimitRanger? how does one affect the other?

Reply:
Proposal: configurations made by the cluster administrators such as [ResourceQuota][k8s-resources-quotas] might not work without LimitRanges. The LimitRanger admission controller can provide default values for resource requests and limits when they have not been defined.

Reply:
I liked the  fgiloux's suggestion.  Done.